modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/index/RocksDbSortedIndexStorage.java

```diff
@@ -171,14 +172,15 @@ private Cursor<ByteBuffer> createScanCursor(byte @Nullable [] lowerBound, byte @
             @Nullable
             private Boolean hasNext;
 
-            @Nullable
-            private byte[] key;
+            private byte @Nullable [] key;
 
             @Override
             public void close() {
-                it.close();
-
-                RocksUtils.closeAll(options, upperBoundSlice);
+                try {
+                    closeAll(it, () -> RocksUtils.closeAll(options, upperBoundSlice));
```

Comment:
Wait a minute ,why not `closeAll(it, options, upperBoundSlice)`? What's the reason for creating a lambda?

Reply:
Fix it